pom.xml

```diff
@@ -0,0 +1,499 @@
+    <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server/blob/main/LICENSE</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Greenbird Integration Technology</name>
+            <email>support@greenbird.com</email>
+            <organization>Greenbird Integration Technology</organization>
+            <organizationUrl>https://www.greenbird.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/utilihive/utilihive-backend-stub-sftp-server.git</connection>
+        <developerConnection>scm:git:ssh://github.com:utilihive/utilihive-backend-stub-sftp-server.git
+        </developerConnection>
+        <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server/tree/main</url>
+    </scm>
+
+    <properties>
+        <annotations.version>23.1.0</annotations.version>
+        <apache-sshd.version>2.9.2</apache-sshd.version>
+        <assertj.version>3.23.1</assertj.version>
+        <commons-io.version>2.11.0</commons-io.version>
+        <dokka.version>1.7.20</dokka.version>
+        <hubspot-dependency-management-plugin.version>0.4</hubspot-dependency-management-plugin.version>
+        <jdk.version>11</jdk.version>
+        <jsch.version>0.2.6</jsch.version>
+        <junit.version>5.9.1</junit.version>
+        <kotlin.version>1.8.0</kotlin.version>
+        <maven-build-info.version>1.3</maven-build-info.version>
+        <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
+        <maven-dependency.version>3.3.0</maven-dependency.version>
+        <maven-deploy-plugin.version>3.0.0</maven-deploy-plugin.version>
+        <maven-detekt.version>1.22.0</maven-detekt.version>
+        <maven-enforcer.version>3.1.0</maven-enforcer.version>
+        <maven-gpg.version>1.6</maven-gpg.version>
+        <maven-install-plugin.version>3.1.0</maven-install-plugin.version>
+        <maven-source.version>3.2.1</maven-source.version>
+        <maven-surefire.version>3.0.0-M7</maven-surefire.version>
+        <memoryfilesystem.version>2.4.0</memoryfilesystem.version>
+        <nexus-staging-maven.version>1.6.7</nexus-staging-maven.version>
+        <slf4j.version>2.0.1</slf4j.version>
+        <sonatype-ossindex-maven-enforcer-rules.version>3.2.0</sonatype-ossindex-maven-enforcer-rules.version>
+
+        <!-- Maven plugins -->
+        <maven-dependency-plugin.skip>false</maven-dependency-plugin.skip>
+        <maven-dependency-plugin.failOnWarning>true</maven-dependency-plugin.failOnWarning>
+        <maven-dependency-plugin.ignoredUnusedDeclaredDependenciesDefaults>
+            org.jetbrains.kotlin:kotlin-stdlib-jdk8
+        </maven-dependency-plugin.ignoredUnusedDeclaredDependenciesDefaults>
```

Comment:
`maven-dependency-plugin.ignoredUnusedDeclaredDependenciesDefaults` can be removed (also from the plugin setting), and `maven-dependency-plugin.runtimeDependencies` can be simplified to this:

```
        <maven-dependency-plugin.runtimeDependencies>
            org.junit.jupiter:junit-jupiter,org.slf4j:slf4j-api,org.slf4j:slf4j-nop
        </maven-dependency-plugin.runtimeDependencies>
```

Reply:
Fixed